CMakeLists.txt

```diff
@@ -38,7 +38,7 @@ set(BUILD_SHARED_LIBS ON CACHE STRING "Build shared libraries")
 
 # Additional targets
 option(ACADOS_UNIT_TESTS "Compile Unit tests" OFF)
-option(ACADOS_EXAMPLES "Compile Examples" OFF)
+option(ACADOS_EXAMPLES "Compile Examples" ON)
```

Comment:
Why do you need to change the default value?

Reply:
Anyway I will be in favor to keep the examples ON by default.
This was off because qpOASES was off by default,
so if you removed all qpOASES usage from the examples,
I would be up for this change.

Reply:
I thought it would be good to compile examples by default. Users would like to start with examples, but if these break the build process they have to figure out why...
Would "Compile Examples" implies qpOASES being enabled be an option?